tests/v3/index.test.ts

```diff
@@ -630,4 +630,78 @@ describe("OpenAPI3 features", () => {
       }`)
     );
   });
+
+  it("empty responses (#333)", () => {
```

Comment:
Nice unit test!

Currently the strategy has been to rely on unit tests as the real core of testing, but the full-schema “snapshots” are just ensuring that this actually works on real-world schemas. Glad you were able to figure it out!

The unit tests themselves could probably be better-organized; I’m not that picky. Open to feedback there. As long as they’re there (like the great ones you added), I’m happy!